_includes/php-dev/composer-types.md

```diff
@@ -6,4 +6,5 @@ The following table discusses the component types that Magento Marketplace suppo
 |Module|magento2-module|Code that modifies Magento application behavior. You can upload a single module to the Magento Marketplace or your module can be dependent on some parent package.|
 |Theme|magento2-theme|Code that modifies the look and feel of the storefront or Magento Admin.|
 |Language package|magento2-language|Translations for the storefront or Admin.|
-
+|Library|magento2-library|Support for libraries located in `lib/internal` instead of in the `vendor` directory.|
+|Component|magento2-component|Good example of `magento2-component` is the [Magento 2 Data Migration Tool]({{ site.mage2bloburl }}/{{ page.guide_version }}/migration/bk-migration-guide.html).|
```

Comment:
Why is it the Migration Tool? It's not a `magento2-component` from what I can see - https://github.com/magento/data-migration-tool/blob/2.3-develop/composer.json
So far the only example is the Magento base package - the package formed of the files that must be located in root (`index.php`, `.htaccess`, etc). This includes `dev/tests` and `setup` as well for now.

Reply:
> Why is it the Migration Tool?

Regarding to https://github.com/magento/magento-composer-installer/blob/master/README.md M2 Migration Tool was developed as a magento-component :)

However, I've checked [MagentoHackathon\Composer\Magento\PackageTypes](https://www.magentoextensions.org/documentation/class_magento_hackathon_1_1_composer_1_1_magento_1_1_package_types.html) and found 
the files that must be located in root .

I've **fixed a provided** description according to your comment. Could you please review?

Thank you, @buskamuza !

Reply:
I see... looks like it's something outdated.
All looks good now.